arm-siterecovery/2015-11-10/swagger/service.json

```diff
@@ -0,0 +1,13707 @@
+      "get": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Gets the details of a vCenter.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "vCenter name of interest.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VCenter"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Perform Add vCenter operation.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "The name of the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "addVCenterRequest",
+            "in": "body",
```

Comment:
Our team had referred these links at the time of development of RestAPIS:
http://sharepoint/sites/AzureUX/Sparta/_layouts/15/WopiFrame.aspx?sourcedoc={CC855351-793A-4417-A536-AB992218AD55}&file=Resource%20Provider%20API%20v2.docx&action=default
and 
http://sharepoint/sites/azure-arc/_layouts/15/WopiFrame.aspx?sourcedoc={e3be54e1-c7a7-4e08-8c12-32670989e599}&action=view&Source=http%3A%2F%2Fsharepoint%2Fsites%2Fazure-arc%2FSitePages%2FREST%2520Guidelines%2Easpx&DefaultItemOpen=1
Also, our RESTAPI model has been reviewed multiple times by ARM team and we never this  feedback.
Having said that, at this point it would be not possible for us to incorporate this request of having same model definition for GET and PUT looking at the impact and efforts needed.